lectures/3-ec.tex

```diff
@@ -176,9 +176,9 @@ \subsubsection{Polynomial Quotient Ring}
 
 \subsubsection{Multiplicative Group of a Finite Field}
 
-The non-zero elements of $\mathbb{F}_p$, denoted as $\mathbb{F}_p^{\times}$, form a multiplicative cyclic group. In other words, there exist elements $f \in \mathbb{F}_p^{\times}$, called \textit{generators}, such that
+The non-zero elements of $\mathbb{F}_p$, denoted as $\mathbb{F}_p^{\times}$, form a multiplicative cyclic group. In other words, there exist elements $g \in \mathbb{F}_p^{\times}$, called \textit{generators}, such that
 \begin{equation}
-    \mathbb{F}_p^{\times} = \{f^k: 0 \leq k \leq p-2\}
+    \mathbb{F}_p^{\times} = \{g^k: 0 \leq k \leq p-2\}
 \end{equation}
 
 The order of $g \in \mathbb{F}_p^{\times}$ is the smallest positive integer $r$ such that $g^r=1$. It is also not difficult to show that $r \mid (q-1)$.
```

Comment:
In that case, we should change $g$ to some other letter here since not necessarily every element of $\mathbb{F}_p^{\times}$ is a generator.

Reply:
Also, change $q$ to $p$, as previously requested by you :)